ledger-core/v2/virtual/object/object.go

```diff
@@ -202,3 +205,38 @@ func (sm *SMObject) stepReadyToWork(ctx smachine.ExecutionContext) smachine.Stat
 func (sm *SMObject) stepWaitIndefinitely(ctx smachine.ExecutionContext) smachine.StateUpdate {
 	return ctx.Sleep().ThenRepeat()
 }
+
+func (sm *SMObject) stepSendVStateReport(ctx smachine.ExecutionContext) smachine.StateUpdate {
+	var (
+		pulseNumber = sm.pulseSlot.CurrentPulseNumber()
+	)
+
+	objDescriptor := sm.Descriptor()
+	prototype, _ := objDescriptor.Prototype()
+	msg := payload.VStateReport{
+		ImmutablePendingCount: int32(sm.ActiveImmutablePendingCount),
```

Comment:
this is incorrect, it should be sum of active and potential

Reply:
Fix pending counting